### DISCUSSION TITLE
Firefly importer with GoCardless only sees 1 bank?

### DISCUSSION THREAD

**Discussion by Quicksoapy** [firefly-iii/firefly-iii#8928] [Firefly III] [answered]
I have added a dutch bank, a revolut account and a paypal account to my GoCardless account. These all seem to function based on the fact that they all show data on the GoCardless website.

When running the importer and using my GoCardless ID and Key only 1 bank shows up, the dutch bank one. This is also the one i first imported.

I have tried redoing the whole importing process, this did not update the available banks.

Does someone know what i could try?

**Comment by JC5** [comment]
What do you mean with "only one bank shows up"?

The country and bank dropdowns in the first step are independent from your configuration. Could you share a screenshot of what the drop-down looks like after you've selected The Netherlands in the data importer?

**Reply by Quicksoapy** [reply]
![image](https://github.com/firefly-iii/firefly-iii/assets/73587977/22f31fa1-6cf2-4ee2-8085-86e2e2008bee)
first, i fill in this data.

![image](https://github.com/firefly-iii/firefly-iii/assets/73587977/b24645bf-0dbe-4b65-9925-e79ece921d91)
Then i choose my bank. This works as expected.

![image](https://github.com/firefly-iii/firefly-iii/assets/73587977/2ba76587-1566-4bf0-9aac-89fc857c08ab)
It then ends up here. I only see this 1 bank account immediately below this drop down. No other accounts are shown.

When opening the Importer again later, it immediately sends me to this page again. I don't know how to add the other banks because of this.

EDIT: i just figured out i can restart the whole process, filling in my GoCardless creds again to select a new bank. I guess this is how i could do it then?

**Reply by JC5** [reply, marked as answer]
Yes, and if you download the configuration files after each import, you can easily recycle them for automation.

**Reply by Quicksoapy** [reply]
Thanks a lot!
